src/v2/Apps/Order/Routes/Review/index.tsx

```diff
@@ -318,6 +319,10 @@ export class ReviewRoute extends Component<ReviewProps> {
 
   render() {
     const { order, isCommittingMutation } = this.props
+    const { artwork } = order.lineItems.edges[0].node
+
+    const isMissingMetadata =
+      !artwork.domesticShippingFee && !artwork.internationalShippingFee
```

Comment:
This is where we previously had the check for `shippingOrigin`. The current implementation covers shipping fee however taxes is not considered here. Does shipping origin somehow contribute to calculating tax? Better question what property would we use to determine if the tax line item is available to display or not or not?

Reply:
I added a check for `taxTotal` here

Reply:
Good catch, changed this to an `||`

Reply:
Now that we have `hasDefiniteTotal` defined in Exchange, we should switch to use that. Something like

```typescript
const isMissingMetadata = order.myLastOffer && !order.myLastOffer.hasDefiniteTotal
```

Probably need to update the relay query to get that field.

Reply:
I thought we made the switch to use the new flag. @lilyfromseattle did our latest changes get pushed?